admin/views/restart/select-restart-list.ejs

```diff
@@ -6,11 +6,164 @@
         <%- partial('partials/_name_and_sign_out') %>
     </div>
 </div>
+<form method="post" action="#">
 
-<div class="grid-row">
+<div class="grid-row" id="select-pupils-restart" aria-describedby="heading-xlarge">
     <div class="column-two-thirds">
-        <header class="page-header">
-            <h1 class="heading-xlarge">Select pupils for restart</h1>
-        </header>
+        <h1 class="heading-xlarge" id="heading-xlarge">Select pupils for restart</h1>
+        <br />
+        <h2>1. Select reason for restart</h2>
+        <ul class="list restart-list">
+            <li>
+                <div class="multiple-choice">
+                    <input id="restart-reason-1"
+                           type="radio"
+                           role="radio"
+                           aria-checked="false"
+                           name="restartReason"
+                           value="Loss of internet">
+                    <label for="restart-code-1">Loss of internet</label>
+                </div>
+            </li>
+            <li>
+                <div class="multiple-choice">
+                    <input id="restart-reason-2"
+                           type="radio"
+                           role="radio"
+                           aria-checked="false"
+                           name="restartReason"
+                           value="Loss of internet">
+                    <label for="restart-reason-2">IT issues</label>
+                </div>
+            </li>
+            <li>
+                <div class="multiple-choice">
+                    <input id="restart-reason-3"
+                           type="radio"
+                           role="radio"
+                           aria-checked="false"
+                           name="restartReason"
+                           value="Loss of internet">
+                    <label for="restart-reason-3">Classroom disruption</label>
+                </div>
+            </li>
+            <li>
+                <div class="multiple-choice">
+                    <input id="restart-reason-4"
+                           type="radio"
+                           role="radio"
+                           aria-checked="false"
+                           name="restartReason"
+                           value="Loss of internet">
+                    <label for="restart-reason-4">Did not complete</label>
+                    <div class="hide">
+                        <div class="panel panel-border-wide">
+                            Please specify:
+                            <textarea class="form-control form-control-3-4" name="did-not-complete-textarea" id="did-not-complete-textarea" rows="1" maxlength="100"></textarea>
+                        </div>
+                    </div>
+                </div>
+            </li>
+        </ul>
+
+        <!--<br />-->
```

Comment:
still required?  Or can this be deleted?

Reply:
removed